code/extensions/SiteTreeSubsites.php

```diff
@@ -208,22 +212,21 @@ public function canPublish($member = null)
 
     /**
      * Create a duplicate of this page and save it to another subsite
-     * @param $subsiteID int|Subsite The Subsite to copy to, or its ID
+	 *
```

Comment:
Wrong indentation, using tabs rather than spaces (looks like we've converted the module to PSR-2 already?)